### PR TITLE
mu 1.8.2

### PR DESCRIPTION
**Pull request by wnka**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/mu.rb [hunk 1]
```diff
@@ -4,10 +4,12 @@
 class Mu < Formula
   desc "Tool for searching e-mail messages stored in the maildir-format"
   homepage "https://www.djcbsoftware.nl/code/mu/"
-  url "https://github.com/djcb/mu/releases/download/1.6.11/mu-1.6.11.tar.xz"
-  sha256 "60eab240dc108f2a419f47d6a75c16841078dcd193f2c0bb02dcdb9ff484ec8d"
+  url "https://github.com/djcb/mu/releases/download/v1.8.2/mu-1.8.2.tar.xz"
+  sha256 "ffac259cd594a10e948f36937fa32e1307aa35259eae9b858c0b92329166533f"
   license "GPL-3.0-or-later"
 
+  head "https://github.com/djcb/mu.git", branch: "master"
```
Comment (chenrui333) [inline Formula/mu.rb:11]:
```suggestion
  head "https://github.com/djcb/mu.git", branch: "master"
```

File: Formula/mu.rb [hunk 2]
```diff
@@ -53,13 +49,15 @@ class Mu < Formula
...
-    system "make", "install"
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+
+    # fix lisp file install location
+    mkdir_p "#{share}/emacs/site-lisp/mu"
+    mv "#{share}/emacs/site-lisp/mu4e", "#{share}/emacs/site-lisp/mu/mu4e"
```
Comment (chenrui333) [inline Formula/mu.rb:60]: any upstream issue tracker for this?

Comment (wnka) [reply Formula/mu.rb:60]: Not sure, this updates the structure to the way it was in previous versions of `mu`.

Comment (wnka) [reply Formula/mu.rb:59]: Created an issue: https://github.com/djcb/mu/issues/2287

File: Formula/mu.rb [hunk 3]
```diff
@@ -53,13 +48,15 @@ class Mu < Formula
...
```
Comment (carlocab) [inline Formula/mu.rb:59]:
```suggestion
    elisp.install share/"emacs/site-lisp/mu4e"
```

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**